app/Files/File.php

```diff
@@ -0,0 +1,2 @@
+<?php
```

Comment:
intent to move upload logic from controller inside here, did half way decide make it work first